doc/source/whatsnew/v1.1.0.rst

```diff
@@ -64,8 +64,12 @@ Other API changes
 Deprecations
 ~~~~~~~~~~~~
 
--
--
+- Passing any arguments but `io` to  :func:`read_html` as positional
```

Comment:
I would maybe say here "any arguments but the first" (since `io` doesn't say much if you don't know the exact names of the signature)

Reply:
can you update this